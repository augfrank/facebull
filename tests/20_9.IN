M0   C0   C1  2436
M1   C0   C2  2020
M2   C0   C3  2023
M3   C0   C4  2560
M4   C0   C5  2674
M5   C0   C6  2738
M6   C0   C7  2727
M7   C0   C8  2908
M8   C0   C9  2733
M9   C0   C10  2319
M10   C0   C11  2918
M11   C0   C12  2940
M12   C0   C13  2588
M13   C0   C14  2004
M14   C0   C15  2234
M15   C0   C16  2494
M16   C0   C17  2165
M17   C0   C18  2509
M18   C0   C19  2411
M19   C1   C0  2302
M20   C1   C2  2713
M21   C1   C3  2368
M22   C1   C4  2557
M23   C1   C5  2083
M24   C1   C6  2366
M25   C1   C7  2467
M26   C1   C8  2854
M27   C1   C9  2043
M28   C1   C10  2300
M29   C1   C11  2106
M30   C1   C12  2450
M31   C1   C13  2174
M32   C1   C14  2571
M33   C1   C15  2689
M34   C1   C16  2277
M35   C1   C17  2477
M36   C1   C18  2098
M37   C1   C19  2541
M38   C2   C0  2362
M39   C2   C1  2801
M40   C2   C3  2122
M41   C2   C4  2778
M42   C2   C5  2800
M43   C2   C6  2754
M44   C2   C7  2679
M45   C2   C8  2929
M46   C2   C9  2717
M47   C2   C10  2701
M48   C2   C11  2858
M49   C2   C12  2334
M50   C2   C13  2576
M51   C2   C14  2611
M52   C2   C15  2220
M53   C2   C16  2920
M54   C2   C17  2791
M55   C2   C18  2238
M56   C2   C19  2440
M57   C3   C0  2514
M58   C3   C1  2156
M59   C3   C2  2323
M60   C3   C4  2705
M61   C3   C5  2021
M62   C3   C6  2554
M63   C3   C7  2251
M64   C3   C8  2632
M65   C3   C9  2820
M66   C3   C10  2570
M67   C3   C11  2736
M68   C3   C12  2278
M69   C3   C13  2571
M70   C3   C14  2874
M71   C3   C15  2652
M72   C3   C16  2892
M73   C3   C17  2522
M74   C3   C18  2581
M75   C3   C19  2169
M76   C4   C0  2698
M77   C4   C1  2276
M78   C4   C2  2822
M79   C4   C3  2025
M80   C4   C5  2673
M81   C4   C6  2325
M82   C4   C7  2600
M83   C4   C8  2218
M84   C4   C9  2167
M85   C4   C10  2291
M86   C4   C11  2542
M87   C4   C12  2410
M88   C4   C13  2701
M89   C4   C14  2010
M90   C4   C15  2725
M91   C4   C16  2996
M92   C4   C17  2668
M93   C4   C18  2767
M94   C4   C19  2818
M95   C5   C0  2379
M96   C5   C1  2303
M97   C5   C2  2995
M98   C5   C3  2297
M99   C5   C4  2898
M100   C5   C6  2975
M101   C5   C7  2531
M102   C5   C8  2849
M103   C5   C9  2432
M104   C5   C10  2047
M105   C5   C11  2628
M106   C5   C12  2384
M107   C5   C13  2849
M108   C5   C14  2399
M109   C5   C15  2733
M110   C5   C16  2855
M111   C5   C17  2078
M112   C5   C18  2707
M113   C5   C19  2094
M114   C6   C0  2829
M115   C6   C1  2130
M116   C6   C2  2841
M117   C6   C3  2938
M118   C6   C4  2397
M119   C6   C5  2237
M120   C6   C7  2373
M121   C6   C8  2102
M122   C6   C9  2035
M123   C6   C10  2139
M124   C6   C11  2849
M125   C6   C12  2622
M126   C6   C13  2663
M127   C6   C14  2285
M128   C6   C15  2337
M129   C6   C16  2345
M130   C6   C17  2992
M131   C6   C18  2257
M132   C6   C19  2919
M133   C7   C0  2225
M134   C7   C1  2571
M135   C7   C2  2614
M136   C7   C3  2702
M137   C7   C4  2886
M138   C7   C5  2248
M139   C7   C6  2289
M140   C7   C8  2211
M141   C7   C9  2105
M142   C7   C10  2574
M143   C7   C11  2251
M144   C7   C12  2472
M145   C7   C13  2046
M146   C7   C14  2159
M147   C7   C15  2493
M148   C7   C16  2787
M149   C7   C17  2310
M150   C7   C18  2988
M151   C7   C19  2949
M152   C8   C0  2249
M153   C8   C1  2676
M154   C8   C2  2132
M155   C8   C3  2980
M156   C8   C4  2863
M157   C8   C5  2047
M158   C8   C6  2786
M159   C8   C7  2985
M160   C8   C9  2174
M161   C8   C10  2898
M162   C8   C11  2448
M163   C8   C12  2335
M164   C8   C13  2634
M165   C8   C14  2462
M166   C8   C15  2350
M167   C8   C16  2064
M168   C8   C17  2179
M169   C8   C18  2833
M170   C8   C19  2828
M171   C9   C0  2977
M172   C9   C1  2268
M173   C9   C2  2278
M174   C9   C3  2429
M175   C9   C4  2529
M176   C9   C5  2061
M177   C9   C6  2193
M178   C9   C7  2600
M179   C9   C8  2070
M180   C9   C10  2639
M181   C9   C11  2911
M182   C9   C12  2066
M183   C9   C13  2970
M184   C9   C14  2223
M185   C9   C15  2999
M186   C9   C16  2977
M187   C9   C17  2670
M188   C9   C18  2760
M189   C9   C19  2684
M190   C10   C0  2734
M191   C10   C1  2524
M192   C10   C2  2899
M193   C10   C3  2859
M194   C10   C4  2797
M195   C10   C5  2041
M196   C10   C6  2768
M197   C10   C7  2780
M198   C10   C8  2332
M199   C10   C9  2707
M200   C10   C11  2128
M201   C10   C12  2322
M202   C10   C13  2940
M203   C10   C14  2565
M204   C10   C15  2537
M205   C10   C16  2025
M206   C10   C17  2713
M207   C10   C18  2188
M208   C10   C19  2372
M209   C11   C0  2666
M210   C11   C1  2961
M211   C11   C2  2508
M212   C11   C3  2034
M213   C11   C4  2263
M214   C11   C5  2428
M215   C11   C6  2834
M216   C11   C7  2365
M217   C11   C8  2875
M218   C11   C9  2503
M219   C11   C10  2938
M220   C11   C12  2392
M221   C11   C13  2382
M222   C11   C14  2951
M223   C11   C15  2410
M224   C11   C16  2442
M225   C11   C17  2328
M226   C11   C18  2602
M227   C11   C19  2046
M228   C12   C0  2424
M229   C12   C1  2803
M230   C12   C2  2736
M231   C12   C3  2312
M232   C12   C4  2199
M233   C12   C5  2684
M234   C12   C6  2358
M235   C12   C7  2047
M236   C12   C8  2158
M237   C12   C9  2447
M238   C12   C10  2032
M239   C12   C11  2275
M240   C12   C13  2576
M241   C12   C14  2368
M242   C12   C15  2369
M243   C12   C16  2120
M244   C12   C17  2620
M245   C12   C18  2602
M246   C12   C19  2114
M247   C13   C0  2964
M248   C13   C1  2062
M249   C13   C2  2531
M250   C13   C3  2174
M251   C13   C4  2848
M252   C13   C5  2309
M253   C13   C6  2608
M254   C13   C7  2057
M255   C13   C8  2998
M256   C13   C9  2179
M257   C13   C10  2390
M258   C13   C11  2742
M259   C13   C12  2240
M260   C13   C14  2515
M261   C13   C15  2490
M262   C13   C16  2893
M263   C13   C17  2701
M264   C13   C18  2202
M265   C13   C19  2488
M266   C14   C0  2674
M267   C14   C1  2527
M268   C14   C2  2494
M269   C14   C3  2472
M270   C14   C4  2019
M271   C14   C5  2959
M272   C14   C6  2856
M273   C14   C7  2960
M274   C14   C8  2002
M275   C14   C9  2745
M276   C14   C10  2004
M277   C14   C11  2701
M278   C14   C12  2799
M279   C14   C13  2321
M280   C14   C15  2043
M281   C14   C16  2763
M282   C14   C17  2375
M283   C14   C18  2756
M284   C14   C19  2607
M285   C15   C0  2311
M286   C15   C1  2013
M287   C15   C2  2076
M288   C15   C3  2879
M289   C15   C4  2806
M290   C15   C5  2913
M291   C15   C6  2486
M292   C15   C7  2337
M293   C15   C8  2141
M294   C15   C9  2039
M295   C15   C10  2496
M296   C15   C11  2930
M297   C15   C12  2178
M298   C15   C13  2386
M299   C15   C14  2733
M300   C15   C16  2876
M301   C15   C17  2189
M302   C15   C18  2845
M303   C15   C19  2801
M304   C16   C0  2389
M305   C16   C1  2353
M306   C16   C2  2198
M307   C16   C3  2474
M308   C16   C4  2302
M309   C16   C5  2051
M310   C16   C6  2696
M311   C16   C7  2275
M312   C16   C8  2764
M313   C16   C9  2292
M314   C16   C10  2059
M315   C16   C11  2662
M316   C16   C12  2472
M317   C16   C13  2318
M318   C16   C14  2821
M319   C16   C15  2385
M320   C16   C17  2653
M321   C16   C18  2989
M322   C16   C19  2109
M323   C17   C0  2823
M324   C17   C1  2098
M325   C17   C2  2346
M326   C17   C3  2382
M327   C17   C4  2354
M328   C17   C5  2674
M329   C17   C6  2681
M330   C17   C7  2087
M331   C17   C8  2317
M332   C17   C9  2957
M333   C17   C10  2636
M334   C17   C11  2578
M335   C17   C12  2261
M336   C17   C13  2957
M337   C17   C14  2883
M338   C17   C15  2708
M339   C17   C16  2628
M340   C17   C18  2951
M341   C17   C19  2452
M342   C18   C0  2656
M343   C18   C1  2222
M344   C18   C2  2160
M345   C18   C3  2276
M346   C18   C4  2190
M347   C18   C5  2569
M348   C18   C6  2322
M349   C18   C7  2137
M350   C18   C8  2888
M351   C18   C9  2565
M352   C18   C10  2791
M353   C18   C11  2132
M354   C18   C12  2458
M355   C18   C13  2360
M356   C18   C14  2655
M357   C18   C15  2654
M358   C18   C16  2704
M359   C18   C17  2424
M360   C18   C19  2621
M361   C19   C0  2657
M362   C19   C1  2115
M363   C19   C2  2825
M364   C19   C3  2911
M365   C19   C4  2469
M366   C19   C5  2489
M367   C19   C6  2620
M368   C19   C7  2353
M369   C19   C8  2001
M370   C19   C9  2901
M371   C19   C10  2352
M372   C19   C11  2353
M373   C19   C12  2509
M374   C19   C13  2457
M375   C19   C14  2642
M376   C19   C15  2506
M377   C19   C16  2103
M378   C19   C17  2207
M379   C19   C18  2109
