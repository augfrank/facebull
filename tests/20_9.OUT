41795
13 29 52 61 79 111 122 145 152 182 205 227 238 264 270 286 306 330 348 369
